### PR TITLE
KRP-1246 Add legal rep name email to business list

### PR DESCRIPTION
**Pull request by p-janik**
Screenshot 2024-12-17 at 10 26 55
Screenshot 2024-12-17 at 10 26 51

### CODE REVIEW

**Review by MikolajKaminski — COMMENTED** [review]
I think the idea was to add the LR not the BO 😅 
Also Dani did ask for email, maybe that requirement has changed? 🤔 

[Link to slack conversation](https://agerascom.slack.com/archives/C049MN99SPL/p1734343712793249?thread_ts=1734340605.023449&cid=C049MN99SPL)

**Comment by p-janik** [comment]
> I think the idea was to add the LR not the BO 😅 Also Dani did ask for email, maybe that requirement has changed? 🤔
> 
> [Link to slack conversation](https://agerascom.slack.com/archives/C049MN99SPL/p1734343712793249?thread_ts=1734340605.023449&cid=C049MN99SPL)

I found the thread later, adjusted

**Comment by p-janik** [comment]
> I think the idea was to add the LR not the BO 😅 Also Dani did ask for email, maybe that requirement has changed? 🤔
> 
> [Link to slack conversation](https://agerascom.slack.com/archives/C049MN99SPL/p1734343712793249?thread_ts=1734340605.023449&cid=C049MN99SPL)

Wanted to use email in the first place, but noticed we are not sending it currently.

**Comment by p-janik** [comment]
@MikolajKaminski please check again 🤞

File: src/routes/backoffice.ts [hunk 1]
```diff
@@ -194,6 +195,26 @@ export const listPersons = async (req, res) => {
...
+          return;
+        }
+
+        const person = await getPerson(
+          legalRepresentative.legal_representative_id
+        );
+        business.meta = business.meta || {};
+        business.meta.email = person.email;
+      } catch (err) {
+        log.error("Error fetching beneficial owner", err);
```
Comment (MikolajKaminski) [inline src/routes/backoffice.ts]:
```suggestion
        log.error("Error fetching person for legal representative", err);
```

**Review by MikolajKaminski — APPROVED** [review]
Small nitpick 😛

**Pull request merged**